pkg/controllers/deprovisioning/drift.go

```diff
@@ -97,9 +96,7 @@ func (d *Drift) ComputeCommand(ctx context.Context, nodes ...*Candidate) (Comman
 			}
 			return Command{}, err
 		}
-		// Log when all pods can't schedule, as the command will get executed immediately.
 		if !results.AllNonPendingPodsScheduled() {
-			logging.FromContext(ctx).With("machine", candidate.NodeClaim.Name, "node", candidate.Node.Name).Debugf("cannot terminate drifted machine since scheduling simulation failed to schedule all pods %s", results.NonPendingPodSchedulingErrors())
```

Comment:
I had seen this log spamming a lot when I was doing testing. For a lot of the time, it had been spamming aggressively with no real impact or information given. I didn't think it was really giving much value, especially as we used this log to alert users when not all pods would reschedule and continue with deprovisioning, but we've since removed that.